AdminDashboard/Models/Domain/Account.cs

```diff
@@ -37,7 +37,9 @@ public EnderUserAccountRequest ToEndUser()
             {
                 Id = Id,
                 Users = Users,
-                CurrentPlan = CurrentPlan.GetPlanType()
+                CurrentPlan = CurrentPlan.GetPlanType(),
+                MaximumNumberOfUsersAllowed = CurrentPlan.MaxNumberOfUsersAllowed,
+                PricePerMonth = CurrentPlan.PricePerMonth
             };
```

Comment:
Change contract for end users - we'll need some other fields from the API. This is so we don't need to do business logic in the client side.